Assets/Editor/OpenAIUtil.cs

```diff
@@ -2,55 +2,78 @@
 using UnityEditor;
 using UnityEngine.Networking;
 
-namespace AICommand {
-
-static class OpenAIUtil
+namespace AICommand
 {
-    static string CreateChatRequestBody(string prompt)
+    public enum ModelType
     {
-        var msg = new OpenAI.RequestMessage();
-        msg.role = "user";
-        msg.content = prompt;
-
-        var req = new OpenAI.Request();
-        req.model = "gpt-3.5-turbo";
-        req.messages = new [] { msg };
+        gpt_4,
+        gpt_3_5_turbo
+    };
 
-        return JsonUtility.ToJson(req);
+    //Dictionary for the ModelType to the string used by the API
+    static class ModelTypeDict
+    {
+        public static string GetModelTypeString(ModelType modelType)
+        {
+            switch (modelType)
+            {
+                case ModelType.gpt_4:
+                    return "gpt-4";
+                default:
```

Comment:
Should be a `ModelType.gpt_3_5_turbo` case before default. More practical for future versions of GPT. I'd also argue the default case should throw an error or at least log that it defaults to 3.5 because the given enum value could not be found.